packages/aura-language-server/package.json

```diff
@@ -31,38 +31,39 @@
     "acorn-loose": "^6.0.0",
     "acorn-walk": "^6.0.0",
     "change-case": "^3.1.0",
-    "enhanced-resolve": "^2.2.2",
-    "fs-extra": "^5.0.0",
+    "enhanced-resolve": "^4.3.0",
+    "fs-extra": "^9.0.1",
     "line-column": "^1.0.2",
-    "minimatch": "^3.0.3",
+    "minimatch": "^3.0.4",
     "properties": "^1.2.1",
-    "resolve-from": "^2.0.0",
+    "resolve-from": "^5.0.0",
     "vscode-languageserver": "^5.2.1",
     "vscode-languageserver-types": "3.14.0",
-    "vscode-nls": "^4.0.0",
+    "vscode-nls": "^4.1.2",
     "vscode-uri": "1.0.6"
   },
   "devDependencies": {
-    "@types/babel-types": "^7.0.4",
-    "@types/fs-extra": "^5.0.4",
-    "@types/glob": "^5.0.35",
-    "@types/jest": "^25.1.2",
+    "@types/babel-types": "^7.0.8",
+    "@types/fs-extra": "^9.0.1",
+    "@types/glob": "^7.1.3",
+    "@types/jest": "^26.0.10",
     "@types/node": "^8.10.0",
-    "@types/shelljs": "^0.8.1",
+    "@types/shelljs": "^0.8.8",
     "babel-types": "^6.26.0",
-    "eslint": "^5.15.3",
-    "glob": "^7.1.1",
-    "husky": "^4.2.3",
-    "jest": "^25.1.0",
-    "lint-staged": "^8.1.0",
+    "eslint": "^7.7.0",
+    "glob": "^7.1.6",
+    "husky": "^4.2.5",
+    "jest": "^26.4.1",
+    "jest-extended": "^0.11.5",
+    "lint-staged": "^10.2.11",
     "npm-run": "^5.0.1",
-    "prettier": "^1.19.1",
-    "shelljs": "^0.8.3",
-    "ts-jest": "^24.3.0",
-    "ts-node": "^7.0.1",
-    "tsc-transpile-only": "^0.0.3",
+    "prettier": "^2.0.5",
+    "shelljs": "^0.8.4",
+    "ts-jest": "^26.2.0",
+    "ts-node": "^8.10.2",
+    "tsc-transpile-only": "0.0.3",
     "tslint": "^6.0.0",
     "tslint-config-prettier": "^1.18.0",
-    "typescript": "^3.1.6"
+    "typescript": "^4.0.2"
```

Comment:
Is there any breaking change between 3 & 4 ? These lang servers are used by the vscode extension which are currently on typescript 3.7.5 (https://github.com/forcedotcom/salesforcedx-vscode/blob/810aba681b2cfabf55a9019ff7f4e6910f57ced1/packages/salesforcedx-vscode-lightning/package.json#L50) and it's important we use compatible versions of typescript.

Reply:
TypeScript doesn't adhere to strict sem versioning - but agreeing to @lcampos's point we should validate TS versions are compatible

Reply:
As we spoke yesterday with Luis, we said that we were let the 3.7.5 version into this packages